### PR TITLE
Grafana weather data dashboard

### PR DESCRIPTION
**Pull request by marijaselakovic**
## Summary of the changes / Why this is an improvement
- Exported JSON file from Grafana weather data dashboard
- Updated README

## Checklist

 - [ ] Link to issue this PR refers to (if applicable): 
 - [ ] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Comment by amotl** [comment]
Thanks for this. Can you also share a screenshot with populated data, run through oxipng?

**Comment by amotl** [comment]
> Should we add this image to the same directory?

Not sure about this. On the other hand, why not? [^1]

[^1]: Having it close to the source means we can also embed into the README here Sure enough it will also be made part of the official documentation about it, and also here, it does not hurt too much when images are referenced on other resources. So, I would be fine with the decision to add images.

File: topic/timeseries/weather-data-grafana-dashboard.json
Comment (amotl) [inline topic/timeseries/weather-data-grafana-dashboard.json:1]: Nit suggestion about naming things and symmetry: `grafana-weather-data-dashboard.json`.

File: topic/timeseries/weather-data-dashboard.png
Comment (amotl) [inline topic/timeseries/weather-data-dashboard.png:1]: Nit suggestion about naming things and symmetry: `grafana-weather-data-dashboard.png`.

**Review by ckurze — APPROVED** [review]
Look good to me - thank you :)

**Pull request merged**